src/app/pages/SearchResultsPage/SearchResultsList.tsx

```diff
@@ -3,7 +3,7 @@ import { useTranslation } from 'react-i18next'
 import { ResultsGroupByType } from './ResultsGroupByType'
 import { RuntimeBlockDetailView } from '../RuntimeBlockDetailPage'
 import { RouteUtils } from '../../utils/route-utils'
-import { TransactionDetailView } from '../TransactionDetailPage'
+import { RuntimeTransactionDetailPage } from '../RuntimeTransactionDetailPage'
```

Comment:
I think we are importing the wrong stuff here. (We need the View, but importing the Page)

Reply:
yep I have updated it in the meantime